openpype/hosts/maya/api/lib.py

```diff
@@ -3655,6 +3655,10 @@ def get_color_management_preferences():
     # Split view and display from view_transform. view_transform comes in
     # format of "{view} ({display})".
     regex = re.compile(r"^(?P<view>.+) \((?P<display>.+)\)$")
+    if int(cmds.about(version=True)) <= 2020:
+        # view_transform comes in format of "{view} {display}" in 2020.
+        regex = re.compile(r"^(?P<view>.+) (?P<display>.+)$")
+
     match = regex.match(data["view_transform"])
```

Comment:
Should we maybe add a check whether the match succeeded here and report a clearer error message than what it previously did?
```python
if not match:
    raise ValueError(
        "Unable to parse view and display from Maya view "
        "transform: '{}' using regex '{}'".format(
            data["view_transform"], regex.pattern
        )
    )
```

Reply:
Sounds good.